### PR TITLE
Update language model to include all necessary metadata for kolibri content rendering

### PR DESCRIPTION
**Pull request by rtibbles**
## Summary

Adds language name and language direction fields to language model, and expands our capacity to deal with longer locale codes.

## Reviewer guidance

Open question - should the language direction constants be in le-utils for use on CC server too?

### CODE REVIEW

**Comment by aronasorman** [comment]
Makes sense that they're in both, because `Language` will eventually come from the cc server anyway. I'll look into integrating [pycountry](https://pypi.python.org/pypi/pycountry) in the future.

**Comment by rtibbles** [comment]
Looks like I need to update a fixture for tests.

**Comment by rtibbles** [comment]
Let's try this one more time without breaking everything this time!

File: kolibri/content/models.py [hunk 1]
```diff
@@ -158,14 +160,15 @@ def get_descendant_kind_counts(self):
 
 @python_2_unicode_compatible
 class Language(ContentDatabaseModel):
-    id = models.CharField(max_length=7, primary_key=True)
+    id = models.CharField(max_length=14, primary_key=True)
     lang_code = models.CharField(max_length=3, db_index=True)
-    lang_subcode = models.CharField(max_length=3, db_index=True, blank=True, null=True)
-
-    objects = ContentQuerySet.as_manager()
```
Comment (ivanistheone) [inline kolibri/content/models.py]: Is the custom `objects` manager not necessary anymore?

Comment (rtibbles) [reply kolibri/content/models.py]: Ugh - I crossed the streams... it soon won't be, but I should reinstate it here.

**Comment by rtibbles** [comment]
Note: should hold off merge until I have fixed the issue @ivanistheone flagged, just for safety.

**Pull request merged**